Core/src/org/sleuthkit/autopsy/casemodule/services/TagsManager.java

```diff
@@ -97,7 +99,17 @@ public static boolean containsIllegalCharacters(String tagDisplayName) {
             /*
              * No current case, nothing more to add to the set.
              */
-        } 
+        }
+        return tagDisplayNames;
+    }
+
+    public static List<String> getNotableTagDisplayNames() {
+        List<String> tagDisplayNames = new ArrayList<>();
+        for (TagNameDefiniton tagDef : TagNameDefiniton.getTagNameDefinitions()) {
+            if (tagDef.isNotable()) {
+                tagDisplayNames.add(tagDef.getDisplayName());
+            }
+        }
```

Comment:
Let's make this a utility method in the centralrepository package rather than expanding the public API here.